js/lib/components/feedback/modal.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from "react";
+import Modal from "react-modal";
+import Highlighter from "web-highlighter";
+
+Modal.setAppElement("body");
+
+const modalStyles = {
+  content: {
+    top: "50%",
+    left: "50%",
+    right: "auto",
+    bottom: "auto",
+    marginRight: "-50%",
+    transform: "translate(-50%, -50%)",
+  },
+};
+
+type FeedbackModalProps = { range: Range; highlighter: Highlighter; toggleModal: () => void };
+const FeedbackModal: React.FC<FeedbackModalProps> = ({ range, highlighter, toggleModal }) => {
+  const [feedback, setFeedback] = useState("");
+
+  const handleFeedbackChange = (e: React.FormEvent<HTMLTextAreaElement>) => {
```

Comment:
Rather than using a state variable, I would just pass a `ref` to the `textarea` and then read `ref.current!.value` inside of `handleSubmit`.